Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -180,9 +180,10 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
         }
 
         bool shouldCapitalize = (desiredType != InGameICChatType.Emote);
+        bool shouldCapitalizeTheWordI = true;
         bool shouldPunctuate = _configurationManager.GetCVar(CCVars.ChatPunctuation);
 
-        message = SanitizeInGameICMessage(source, message, out var emoteStr, shouldCapitalize, shouldPunctuate);
+        message = SanitizeInGameICMessage(source, message, out var emoteStr, shouldCapitalize, shouldCapitalizeTheWordI, shouldPunctuate);
```

Comment:
Please no more args I beg.

Reply:
Not really any way I can reduce the number, unfortunately. Capitalizing I is only a thing done in English, so it needs to be togglable.